### PR TITLE
tour service abstraction

### PR DESCRIPTION
**Pull request by grigor-yeghiazaryan**

### CODE REVIEW

File: src/TourManager/TourManager.Service.Model/Tour.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace TourManager.Service.Model
+{
+	public class Tour
+	{
+		public int Id { get; set; }
+		public int TenantId { get; set; }
+
```
Comment (TomaCode) [inline src/TourManager/TourManager.Service.Model/Tour.cs:10]: empty line, but you can ignore it. we'll fix in future

File: src/TourManager/TourManager.Service.Abstraction/ITourService.cs [hunk 2]
```diff
@@ -0,0 +1,15 @@
+﻿using System.Threading.Tasks;
+using TourManager.Service.Model;
+using System.Collections.Generic;
+
+namespace TourManager.Service.Abstraction
+{
+    public interface ITourService
```
Comment (TomaCode) [inline src/TourManager/TourManager.Service.Abstraction/ITourService.cs:7]: we also need IClientService, IPropertyService I guess

**Review by TomaCode — APPROVED** [review]
not all the service abstractions are defined, but fine for me. you can continue later

**Review by HAghajanyan-ST — APPROVED** [review]

**Review by HAghajanyan-ST — APPROVED** [review]

File: src/TourManager/TourManager.Service.Abstraction/ITourService.cs (hunk 2 above)
Comment (grigor-yeghiazaryan) [reply src/TourManager/TourManager.Service.Abstraction/ITourService.cs:7]: yep

**Pull request merged**